components/badge/__tests__/badge.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
```

Comment:
Can you rename the file to `badge.test.tsx` please?

Reply:
feel free to do it yourself 👍 (because I am OOO for today ;))

Reply:
I will do, thanks.